tools/Makefile.shared

```diff
@@ -202,7 +216,27 @@ addlabels: addlabels.cmo
 		$(ADDLABELS_IMPORTS) addlabels.cmo
 
 #install::
-#	cp addlabels "$(INSTALL_LIBDIR)"
+#	cp addlabels "$${INSTALL_LIBDIR}"
+
+ifeq '$(EXE)' ''
```

Comment:
It's not a good idea to use `EXE` to detect Windows. You should probably look at `SYSTEM` instead.
In fact, under Cygwin we have `EXE=.exe` but symbolic links are supported.